test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1146,6 +1146,100 @@ public void PcaOnnxConversionTest()
             Done();
         }
 
+        private class HashData
+        {
+            public uint Data { get; set; }
+        }
+
+        [Fact]
+        public void MurmurHashKeyTest()
```

Comment:
>MurmurHashKeyTest [](start = 20, length = 17)

Shouldn't we have both `MurmurHashStringTest` and `MurmurHashKeyTest`?

Reply:
I added the String test to MurmurHashTest to make more concise. Key needs its own test, since ValueToKey needs to be added to the pipeline.